jenkins_build.sh

```diff
@@ -0,0 +1,20 @@
+set -e
```

Comment:
This is a pretty common shell script trick to add the equivalent of:

if [[ $? -ne 0 ]]; then exit 1; fi

Think of it as implicitly adding the `if err != nil` check in Go.